docs/fleet/cli.md

```diff
@@ -686,6 +695,57 @@ See [Upload a custom module](/registry/upload/#upload-a-custom-module) and [Upda
 
 See [Modular resources](/registry/) for a conceptual overview of modules and the modular resource system at Viam.
 
+##### Using the `build` subcommand
+
+You can use the `module build start` or `module build local` commands to build your custom module according to the build steps you specify in your <file>meta.json</file> file.
+
+You could use `build start` to build or compile your module on a cloud build host that might offer additional platform support than you have access to locally, or you could use `build local` to quickly test that your module builds or compiles as expected on your local hardware.
+
+To configure your module's build steps, add a `build` object to your [`meta.json` file](#the-metajson-file), resembling the following:
+
+```json {class="line-numbers linkable-line-numbers"}
+"build": {
+  "setup": "setup.sh",                    // optional - command to install your build dependencies
+  "build": "make module.tar.gz",          // command that will build your module
+  "path" : "module.tar.gz",               // optional - path to your built module
+                                          // (passed to the 'viam module upload' command)
+  "arch" : ["linux/amd64", "linux/arm64"] // architecture(s) to build for
+}
+```
+
+For example, the following extends the `my-module` <file>meta.json</file> file from the previous section with a new `build` object to control its build parameters when used with `module build start` or `module build local`:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "module_id": "acme:my-module",
+  "visibility": "public",
+  "url": "https://github.com/acme-co-example/my-module",
+  "description": "An example custom module.",
+  "models": [
+    {
+      "api": "rdk:component:generic",
+      "model": "acme:demo:my-model"
+    }
+  ],
+  "build": {
+    "setup": "setup.sh",
+    "build": "make module.tar.gz",
+    "path": "module.tar.gz",
+    "arch": ["linux/amd64", "linux/arm64"]
+  },
+  "entrypoint": "run.sh"
+}
+```
+
+When you initiate a build job using either `start` or `local`, the command returns the build ID of your job.
+You can then provide that build ID to the `module build logs` command to show the relevant build logs for that build.
```

Comment:
We should mention the `module build list` command here. It will show the build they just started and the status for all the builds